themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -50,7 +50,7 @@
 
     {block name='product_unit_price'}
       {if $displayUnitPrice}
-        <p class="product-unit-price">{convertPrice price=$unit_price} {l s='per %s' sprintf=$product.unity}</p>
+        <p class="product-unit-price"{hook h='displayProductPriceBlock' product=$product type="unit_price" returntype="style"}>{convertPrice price=$unit_price} {l s='per %s' sprintf=$product.unity}</p>
```

Comment:
@Shudrum 
OK my PR here is now up to date and I also submitted the corresponding commit for the hook in ps-legalcompliance